doc/tools/docgram/orderedGrammar

```diff
@@ -1295,8 +1295,8 @@ coercion_class: [
 ]
 
 syntax_modifier: [
-| "at" "level" natural
-| "in" "custom" ident OPT ( "at" "level" natural )
+| "at" [ "level" natural | "existing" "level" OPT ( "or" natural ) ]
+| "in" "custom" ident OPT ( "at" [ "level" natural | "existing" "level" OPT ( "or" natural ) ] )
 | LIST1 ident SEP "," [ "at" level | "in" "scope" ident ]
 | ident "at" level OPT binder_interp
 | ident explicit_subentry
```

Comment:
We try to list productions from simplest or most common to most complex to least common.  This is subjective.  On the face of it, this order seems to have suffered some entropy.  Perhaps you could put the entries for `syntax_modifier` into a more useful order?

Reply:
Well, I'm no expert but the order still feels reasonnably consistent to me, in the sense that what looks to me like the most basic options come first. I only fixed a minor quirk by the end.